alt-fuel-finder/spec/models/station_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'rails_helper'
+
+RSpec.describe Station, type: :model do
+  context "#nearest_stations_by_zip" do
+    it "should return station objects" do
+      zip_code = 80203
+      stations = Station.nearest_stations_by_zip(zip_code)
+      station = stations.first
+  
+      expect(station).to be_kind_of(Struct::Station)
```

Comment:
@josh-works Yeah, that's a good point. I think you should generally follow your ❤️ . So if DDDing would lead you a certain way, most of the time that's what you should do. I'm guessing you would end up using a PORO instead of `Struct`.